pkg/controllers/management/auth/common/common.go

```diff
@@ -0,0 +1,10 @@
+package common
```

Comment:
Unfortunately this is used by both the `role_template_lifecycle.go` file and the `globalroles` package, so I can't put it in `auth` which imports `globalroles` - I suspect that this will become a circular import.

Besides I think that this package makes sense if we continue the pattern that this PR starts with the other rbac packages (i.e. one package for GR/GRBs, one for CRTBs, one for PRTBs, one for RoleTemplates) - there will be common overlap with things like subject resolution.

In short, I understand the concern but plan to leave as-is.

Reply:
Could the package be named differently?

Reply:
I can't really think of a better name - this accurately describes the purpose of the package. I'm open to suggestions if you have any.

Reply:
`common` just means nothing, perhaps `errors` would be better despite the collision with the stdlib package.

Reply:
If you expect other things to go there, then you might want to add more packages or sub-packages for them.
It's no big deal,`common`  just doesn't improve readability when you see a call like `common.IsClusterUnavailable` - I've no idea what common might mean here.
At least the file could be called errors or something like that.
Up to you...

Reply:
Resolving, I've moved this to the clustermanager package (which is what can produce these errors) instead.